launcher/launcher.go

```diff
@@ -1426,3 +1430,55 @@ func (launcher *Launcher) stopRunningInstances() error {
 
 	return nil
 }
+
+func (launcher *Launcher) removeOutdatedInstances() error {
+	log.Debug("Remove outdated instances")
+
+	instances, err := launcher.storage.GetAllInstances()
+	if err != nil {
+		return aoserrors.Wrap(err)
+	}
+
+	launcher.getCurrentServices(instances)
```

Comment:
In my opinion a little misunderstanding method naming

Reply:
renamed to cacheCurrentServices